samples/openapi3/client/petstore/typescript/builds/explode-query/models/Capitalization.ts

```diff
@@ -19,7 +19,8 @@ export class Capitalization {
     'capitalSnake'?: string;
     'sCAETHFlowPoints'?: string;
     /**
-    * Name of the pet 
+    * Name of the pet
+
```

Comment:
I didn't add trimming to the `escapeUnsafeCharacters` implementation. `escapeText` has it, hence why this now has an additional newline. Let me know if you prefer trimming leading and trailing whitespace for this and if yes, whether you prefer it for the Typescript generator in specific or generally for all implementations.